pro/src/pages/CollectiveOfferPreviewCreation/__specs__/CollectiveOfferPreviewCreation.spec.tsx

```diff
@@ -66,4 +73,34 @@ describe('CollectiveOfferPreviewCreation', () => {
       })
     ).toBeInTheDocument()
   })
+
+  it('Should show the redirect modal', async () => {
```

Comment:
(nitpick) *should en minuscule (pour que ça se lise comme une phrase, it should...

Reply:
Bien vu, je les ai tous corrigés au passage